helix-term/src/commands.rs

```diff
@@ -4541,14 +4541,18 @@ fn shell_impl(
     use std::process::{Command, Stdio};
     ensure!(!shell.is_empty(), "No shell set");
 
-    let mut process = match Command::new(&shell[0])
+    let mut process_ = Command::new(&shell[0]);
+    process_
```

Comment:
No need for the trailing `_` in `process_`. The `process` binding below will [shadow](https://doc.rust-lang.org/rust-by-example/variable_bindings/scope.html) this one

Elsewhere we've used `variable_` to avoid variable names that are keywords like [`match_`](https://github.com/helix-editor/helix/blob/0c08ff1596b398d53c8c48cc96cfbd24ddec303b/helix-core/src/syntax.rs#L1623)

Reply:
Thanks. Sorry for the mess, still in the process of learning rust!